samples/bluetooth/peripheral_lbs/prj_minimal.conf

```diff
@@ -107,5 +107,6 @@ CONFIG_BT_BUF_EVT_RX_COUNT=2
 CONFIG_BT_CONN_TX_MAX=3
 CONFIG_BT_L2CAP_TX_BUF_COUNT=2
 CONFIG_BT_ATT_TX_COUNT=2
+CONFIG_BT_BUF_EVT_RX_COUNT=4
```

Comment:
what is the source of these values?

Reply:
@weeTike ptal

Reply:
The source of the value is CONFIG_BT_BUF_ACL_TX_COUNT + 1. This is needed due to this change https://github.com/zephyrproject-rtos/zephyr/commit/d382fca6ff8830fff33cc81097d724e1314af92b#diff-b199a522335b761f8d05548154babf2722b207853be7e7e02cfa8eb21913bf9a